### ISSUE TITLE
question about flow loss

### ISSUE THREAD

**Issue by GreatEngineer** [fjchange/pytorch_ano_pre#6]
I see that the default weight of flow loss is 0, which means no flow loss. Does it means when there is no flow loss, it can have a result of 85.6%? So how many performance improvement of the litenet?

**Comment by fjchange** [comment]
Actually, LiteFlownet without finetuned on SmallMotion can not boost the perfomance, but the way to calculate score I modified help actually. With Flownet2,  the result can be even better, which have been added in the latest version.

**Comment by GreatEngineer** [comment]
thanks for your reply, and I notice that, the way to calculate score you modified can help in avenue, but for other dataset like ped2 or shanghaitech, the performance seems would drop a lot.... Did you perform an experiment?

**Comment by fjchange** [comment]
I think the one without normalization is more reasonable. If we normalize the score for each video which treat each video in testing set as anomaly video ( they are anomaly video actually ), it would will boost the testing result but not partical for application.
